src/resources/taxjar-credentials-resource.js

```diff
@@ -0,0 +1,27 @@
+export default function TaxJarCredentialsResource({apiHandler}) {
```

Comment:
I miss the "autogenerated header" in this file. Was it manually generated?

We should use [rebilly-sdk-generator](https://github.com/Rebilly/rebilly-sdk-generator/blob/f94eb73b9316978e9b92785dda2f198007f3eb70/package.json#L17) instead.

Reply:
It does not generate the methods for tax-jar, is there something wrong with API definitions?